docs/pages/deploy-a-cluster/helm-deployments/custom.mdx

```diff
@@ -23,67 +31,71 @@ migrating your setup from a legacy version of the Helm charts.
-  enabled: true
-  listen_addr: 0.0.0.0:3080
-  public_addr: custom.example.com:443
-
-ssh_service:
-  enabled: true
-  labels:
-    cluster: custom
-  commands:
-  - name: kernel
-    command: [/bin/uname, -r]
-    period: 5m
-EOF
-```
+- The `proxy` pod configuration should contain at least the `proxy_service` section and the `teleport` section without the `storage` part.
+- The `auth` pod configuration should contain at least the `auth_service` section and the `teleport` one.
 
-<Admonition type="tip">
-  You can skip this step if you already have a `teleport.yaml` file locally that you'd like to use.
-</Admonition>
+The chart automatically creates a Kubernetes join token named after the Helm release which will enable the proxy pods to seamlessly connect to the auth pods.
+If you do not want to use this automatic token, you must provide a valid Teleport join token in the proxy pods' configuration.
 
-Create the namespace for the config and add the `teleport.yaml` from your local
-disk:
+<Admonition type="warning">
+  When using `scratch` or `standalone` mode, you **must** use highly-available storage (e.g. etcd, DynamoDB, or Firestore) for multiple replicas to be supported.
 
-```code
-$ kubectl create namespace teleport
-$ kubectl --namespace teleport create configmap teleport --from-file=teleport.yaml
-```
+  [Information on supported Teleport storage backends](../../reference/backends.mdx)
 
-<Admonition type="note">
-  The name of the `ConfigMap` used must match the name of the Helm release that you install below (the name just after `helm install`).
-  In this example, it's `teleport`.
+  Manually configuring NFS-based storage or `ReadWriteMany` volume claims is **NOT** supported for an HA deployment and will result in errors.
+</Admonition>
 
-  The name (key) of the configuration file uploaded to your `ConfigMap` must be `teleport.yaml`. If your configuration file is named differently
-  on disk, you can specify the key that should be used in the `kubectl` command:
+Write the following `my-values.yaml` file, and adapt the teleport configuration as you want.
+You can find all possible configuration fields in the [Teleport Config Reference](../../reference/config.mdx).
+
+```yaml
+chartMode: scratch
+
+auth:
+  teleportConfig:
+    # put your teleport.yaml auth configuration here
+    teleport:
+      log:
+        output: stderr
+        severity: INFO
+
+    auth_service:
+      enabled: true
+      listen_addr: 0.0.0.0:3025
+
+proxy:
+  teleportConfig:
+    # put your teleport.yaml proxy configuration here
+    teleport:
+      # The join_params section must be provided for the proxies to join the auth servers
+      # By default, the chart creates a Kubernetes join token which you can use.
+      join_params:
+        method: kubernetes
+        token_name: "RELEASE-NAME-proxy"  # replace RELEASE-NAME
+      auth_server: "RELEASE-NAME-auth.RELEASE-NAMESPACE.svc.cluster.local:3025"  # replace RELEASE-NAME and RELEASE-NAMESPACE
```

Comment:
Oh, there's a vocabulary conflict here. Helm also refers to a specific deployment of an application as a Release (because the term Deployment is coined in Kubernetes).

I'll define the RELEASE-NAME and RELEASE-NAMESPACE placeholders before the code block to remove the confusion.

I would love to have a way to mark part of the code snippets as placeholders, like what google is doing with the pink-italic-uppercase placeholders in their docs.

Reply:
🎉  this is exactly what I was looking for, thank you!